guides/common/modules/proc_exporting-templates.adoc

```diff
@@ -15,24 +15,31 @@ The {ProjectWebUI} displays the status of the export.
 The status is not persistent; if you leave the status page, you cannot return to it.
 
 .CLI procedure
-. Clone a local copy of your Git repository:
+. Clone a local copy of your Git repository to a subdirectory of `/var/lib/foreman`:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-$ git clone https://github.com/theforeman/community-templates __/custom/templates__
+$ git clone https://github.com/theforeman/community-templates /var/lib/foreman/community-templates
 ----
-. Change the owner of your local directory to the `foreman` user, and change the SELinux context with the following commands:
++
+[NOTE]
+====
+You can clone your repository to a custom directory outside `/var/lib/foreman`, but you have to ensure that the `Foreman` service can read its contents.
+The directory must have the correct file permissions and the `foreman_lib_t` SELinux label.
+====
+. Change the owner of your local directory to the `foreman` user:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-# chown -R foreman:foreman __/custom/templates__
-# chcon -R -t httpd_sys_rw_content_t __/custom/templates__
+# chown -R foreman:foreman /var/lib/foreman/community-templates
 ----
 . To export the templates to your local repository, enter the following command:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-hammer export-templates --organization 'Default Organization' --repo __/custom/templates__
+hammer export-templates \
+--organization 'Default Organization' \
```

Comment:
```suggestion
--organization "Default Organization" \
```
In some places, we use double quotes. Not sure which one to follow.

Reply:
Fixed.